socialdistribution/templates/posts/posts_form.html

```diff
@@ -25,4 +30,22 @@
     </div>
   </div>
 </div>
+<script type="text/javascript">
+  $(document).ready(function(){
+    $(".fa-camera").hide();
+    $("#id_image_file").hide();
+    $('#id_content_type').on('change', function() {
+      if ( this.value == 'image/png;base64' || this.value == 'image/jpeg;base64'){
+        $("#id_content").attr("required", false)
+        $("#id_content").hide();
+        $(".fa-camera").show();
+        $("#id_image_file").show();
+      } else {
+        $(".fa-camera").hide();
+        $("#id_image_file").hide();
+        $("#id_content").show();
+      }
+    });
+  });
+</script>
```

Comment:
I think there is a subfile `static/js` folder. If you could put the javascript in those folder, the html only for html.

Reply:
add `{% load staticfiles %}` at the top of the template or `{% load static %}`  will work.
For the head section refer to script such as`<script src="{% static "MyApp1/js/scriptForMyApp1.js" %}"></script>`
https://www.reddit.com/r/django/comments/4f7v1n/how_do_i_include_javascript_in_my_django_project/

Reply:
Thanks, changes are made